specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -60,14 +60,16 @@ export function RecordMergingLink({
   const [records] = useSearchParameter(mergingQueryParameter, overlayLocation);
   const oldRecords = React.useRef(records);
   const needUpdateQueryResults = React.useRef(false);
+
   React.useEffect(() => {
     if (oldRecords.current === undefined && records !== undefined)
       needUpdateQueryResults.current = false;
     // Detect agent merging dialog getting closed after some records are merged
     else if (
       records === undefined &&
-      oldRecords.current !== undefined &&
-      needUpdateQueryResults.current
+      oldRecords.current !== undefined
+      // &&
+      // needUpdateQueryResults.current
```

Comment:
is this not working correctly?
do you need help, or just time to work on this?

sorry for bothering you about this, just checking in...